packages/oauth/src/index.ts

```diff
@@ -60,7 +63,10 @@ export class InstallProvider {
     } else {
       this.logger = getLogger('OAuth:InstallProvider', logLevel ?? LogLevel.INFO, logger);
     }
-
+    this.stateValidation = stateValidation;
+    if (!stateValidation) {
+      this.logger.info('You\'ve disabled mandatory state validation during oauth. When state is provided as a query param we will attempt to validate, when it is omitted, with this flag set to true, we will ignore validation. Custom metadata options will not be available when state is not supplied.');
```

Comment:
How about using double-quotes? With the way, you don' need to escape single-quotes.

Reply:
s/oauth/OAuth/

Reply:
>with this flag set to true, 

Perhaps, the log message readers do not understand this part (What's "this flag"???). We can be more specific. If I try to write the message, I will go with something like this:

You've disabled OAuth's state parameter validation by setting `InstallProvider#stateValidation` to false. The option is supposed to be used only for Org-Wide app installations from admin pages. If this is not the case, we highly recommend setting the flag to true and starting your OAuth flow from `/slack/install` or your own starting endpoint. The state parameter can protect your end-users from CSRF attacks. Refer to https://datatracker.ietf.org/doc/html/rfc6749#section-10.12 for more details.

I know this is too long. We can simplify the sentence, though.

Reply:
Good point, the wording is a bit confusing at the moment. Let me reword with your feedback in mind and see whether we can make it shorter. I will also add a section to SDK documentation.

Reply:
Thanks!